appdatabase/migrations/sql/1683627613_accounts_and_keycards_improvements.up.sql

```diff
@@ -0,0 +1,94 @@
+-- Possible `operable` column values:
+-- "no"        // an account is non operable it is not a keycard account and there is no keystore file for it and no keystore file for the address it is derived from
+-- "partially" // an account is partially operable if it is not a keycard account and there is created keystore file for the address it is derived from
+-- "fully"     // an account is fully operable if it is not a keycard account and there is a keystore file for it
+
+-- Adding new tables `keypairs` and `keypairs_accounts`
+CREATE TABLE IF NOT EXISTS keypairs (
```

Comment:
after we added table keypairs, when user login, there should be/must exist only 1 record which's type is `KeypairTypeProfile` and we can have many records other than `KeypairTypeProfile`, correct?

Reply:
Yes, there will be only a single keypair marke as "profile" and there may be 0 to N keypairs of type "seed" or "key".